scripts/11_retrieve_profiles.py

```diff
@@ -58,8 +58,36 @@
 
 # %%
 profile_index = pl.read_csv(INDEX_FILE)
+
 profile_index
 
+# %% [markdown]
+# ## Pipeline Information Encoded in File Names
```

Comment:
@afermg I think a note like this will be helpful too

It demystifies the names

Reply:
https://github.com/broadinstitute/jump_hub/pull/99#pullrequestreview-2960383517 approved